heron/scheduler-core/src/java/com/twitter/heron/scheduler/UpdateTopologyManager.java

```diff
@@ -163,10 +163,6 @@ private void updateTopology(final PackingPlans.PackingPlan existingProtoPackingP
       scalableScheduler.get().addContainers(containerDelta.getContainersToAdd());
     }
 
-    // update parallelism in updatedTopology since TMaster checks that
-    // Sum(parallelism) == Sum(instances)
-    logInfo("Update new Topology: %s", stateManager.updateTopology(updatedTopology, topologyName));
```

Comment:
If we remove the topology-update on stateManager, it seems we could remove or clean a lot of codes, like:
```
private TopologyAPI.Topology getUpdatedTopology(...)
static TopologyAPI.Topology mergeTopology(...)
private static boolean updateComponent(...)
```
And a lot of unit tests.
Can we also do that?

Reply:
removed